src/datashare/HISTORY.rst

```diff
@@ -1,13 +1,8 @@
-.. :changelog:
-
-Release History
-===============
-
-0.1.1
-+++++
-* Add examples for creating a BlobFolder dataset
-
-
-0.1.0
-++++++
-* Initial release.
+.. :changelog:
+
+Release History
+===============
+
+0.1.0
+++++++
+* Initial release.
```

Comment:
Is this codegen's bug to overwrite history notes? @kairu-ms

Reply:
Yes, That's know issue.

Reply:
@hsrivast Please append new version info instead of removing all history notes